src/vcpkg/build.cpp

```diff
@@ -1315,22 +1315,26 @@ namespace vcpkg::Build
         return Strings::format("Error: Building package %s failed with: %s", spec, Build::to_string(build_result));
     }
 
-    std::string create_user_troubleshooting_message(const PackageSpec& spec)
+    std::string create_user_troubleshooting_message(const PackageSpec& spec, const VcpkgPaths& paths)
     {
 #if defined(_WIN32)
         auto vcpkg_update_cmd = ".\\vcpkg";
 #else
         auto vcpkg_update_cmd = "./vcpkg";
 #endif
-        return Strings::format("Please ensure you're using the latest portfiles with `%s update`, then\n"
+        auto description = paths.git_describe_head();
+        return Strings::format("Please ensure you're using the latest portfiles with `git pull` and `%s update`, then\n"
                                "submit an issue at https://github.com/Microsoft/vcpkg/issues including:\n"
                                "  Package: %s\n"
+                               "  Vcpkg-tool version: %s\n"
```

Comment:
Also, this isn't _really_ related to the PR, but this should be (and the one below) `s/Vcpkg/vcpkg/`

Reply:
I wanted to be consistent with the other lines. Should I then also change `Package` to `package`

Reply:
If you like; `vcpkg` is the correct title case for the vcpkg name. it should never be capitalized, even at the front of a sentence or in a title.